application/core/pipeline.py

```diff
@@ -97,20 +95,23 @@ def fetch_response_data(
         os.makedirs(os.path.join(column_field_dir, dataset), exist_ok=True)
         os.makedirs(os.path.join(dataset_resource_dir, dataset), exist_ok=True)
 
-        pipeline_run(
-            dataset=dataset,
-            pipeline=pipeline,
-            specification_dir=specification_dir,
-            input_path=file_path,
-            output_path=os.path.join(transformed_dir, dataset, f"{file_name}.csv"),
-            issue_dir=os.path.join(issue_dir, dataset),
-            column_field_dir=os.path.join(column_field_dir, dataset),
-            dataset_resource_dir=os.path.join(dataset_resource_dir, dataset),
-            organisation_path=os.path.join(cache_dir, "organisation.csv"),
-            save_harmonised=False,
-            organisations=[organisation],
-            custom_temp_dir=os.path.join(cache_dir),
-        )
+        try:
```

Comment:
When this errors do we have a specific response that is given back to the clent?

Reply:
No it is mostly for logging purpose so it is easy for us to check if something goes wrong